radixdlt/src/main/java/com/radixdlt/consensus/SyncedStateComputer.java

```diff
@@ -38,6 +38,12 @@
 		 * @return the BFT data for the instruction
 		 */
 		VertexMetadata getVertexMetadata();
+
+		/**
+		 * Retrieve the BFT timestamp for the instruction
+		 * @return the BFT timestamp
+		 */
+		long getTimestamp();
```

Comment:
I think this should be part of VertexMetadata since it is metadata created by BFT